### PR TITLE
Add recommended pod count

### PR DESCRIPTION
**Pull request by zazaphone**

### CODE REVIEW

File: appuio-specifics-public.rst [hunk 1]
```diff
@@ -50,6 +50,12 @@ Quotas and Limits
 The quotas are defined in the project size you ordered. The exact numbers can be found
 on the product site `APPUiO Public Platform <https://appuio.ch/public.html>`__
 
+Pod count Recommendations
+------------------------
+
+Red Hat supports 250 pods per node. We do not recommend this many pods per node as this can lead to stability issues. We recommend 100 pods per node to avoid any stability issues.
```
Comment (mhutter) [inline appuio-specifics-public.rst]: Upstream Kubernetes documentation says max 100 Pods per node are supported, maybe it is worth linking here?

Comment (megian) [reply appuio-specifics-public.rst]: 100 and 250 are a maximum of pods depending on the worklod

**Review by megian — APPROVED** [review]

File: appuio-specifics-public.rst [hunk 2]
```diff
@@ -50,6 +50,13 @@ Quotas and Limits
 The quotas are defined in the project size you ordered. The exact numbers can be found
 on the product site `APPUiO Public Platform <https://appuio.ch/public.html>`__
 
+Pod count Recommendations
+-------------------------
+The technical Red Hat limit is 250 pods per node. Kubernetes however recommends a maximum of 100 pods per node. 
+The maximum pod count per node is dependent on the workload. We recommend a maximum of 100 pods per node as you may face stability issues with a higher number of pods.
+
+For further information on the Kubernetes and Red Hat pod limits please check the official `Kubernetes <https://kubernetes.io/docs/setup/best-practices/cluster-large/.html>`__ and `Red Hat <https://docs.openshift.com/container-platform/3.7/scaling_performance/cluster_limits.html>`__ documentation.
```
Comment (megian) [inline appuio-specifics-public.rst]: Something went wrong here with the link `.html` and OpenShift 3.7 is outdated! Please use 3.11!

File: appuio-specifics-public.rst [hunk 3]
```diff
@@ -50,6 +50,13 @@ Quotas and Limits
 The quotas are defined in the project size you ordered. The exact numbers can be found
 on the product site `APPUiO Public Platform <https://appuio.ch/public.html>`__
 
+Pod count Recommendations
+-------------------------
+The technical Red Hat limit is 250 pods per node. Kubernetes however recommends a maximum of 100 pods per node. 
+The maximum pod count per node is dependent on the workload. We recommend a maximum of 100 pods per node as you may face stability issues with a higher number of pods.
+
+For further information on the Kubernetes and Red Hat pod limits please check the official `Kubernetes <https://kubernetes.io/docs/setup/best-practices/cluster-large/.html>`__ and `Red Hat <https://docs.openshift.com/container-platform/3.11/scaling_performance/cluster_maximums.html#scaling-performance-current-cluster-maximums.html>`__ documentation.
```
Comment (megian) [inline appuio-specifics-public.rst]: `https://kubernetes.io/docs/setup/best-practices/cluster-large/.html` should be `https://kubernetes.io/docs/setup/best-practices/cluster-large/` right?

**Review by mhutter — APPROVED** [review]

**Pull request merged**